acestep/api_server.py

```
@@ -2806,6 +3097,1607 @@ async def format_input_endpoint(request: Request, authorization: Optional[str] =
+            raise HTTPException(status_code=400, detail="No dataset loaded. Please scan or load a dataset first.")
+
+        handler: AceStepHandler = app.state.handler
+        llm: LLMHandler = app.state.llm_handler
+
+        if handler is None or handler.model is None:
+            raise HTTPException(status_code=500, detail="Model not initialized")
+
+        if llm is None or not llm.llm_initialized:
+            raise HTTPException(status_code=500, detail="LLM not initialized")
+
+        mgr = _RuntimeComponentManager(handler=handler, llm=llm, app_state=app.state)
```

Ruff (0.15.0)

[error] Undefined name `_RuntimeComponentManager`

(F821)